### DISCUSSION TITLE
That time you forgot to disable global build permissions in your gathertown:

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#52] [Team Posts]
<img width="784" alt="Screen Shot 2021-07-23 at 8 13 34 AM" src="https://user-images.githubusercontent.com/10963114/126811539-dec0a8b0-d5f5-499a-b77d-99fb64fbae7a.png">

(Glenn accidentally got stuck in build mode (for which the default object is that plant) and kept leaving trees behind him wherever he went. And couldn't understand what was happening. It was absolutely hilarious.)

**Comment by glenn-slayden** [comment]
It turned out to be a packed forest, as there were several trees hidden underneath each one shown…

**Comment by fcbond** [comment]
Glenn appleseed!

On Sat, Jul 24, 2021 at 12:37 AM Glenn Slayden ***@***.***>
wrote:

> It turned out to be a packed forest, as there were several trees hidden
> underneath each one shown…
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/teams/participants/discussions/6/comments/1>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRXKPU4NTHE3I7NFTPLTZGLEJANCNFSM5A4JMF7Q>
> .
>


-- 
Francis Bond <http://www3.ntu.edu.sg/home/fcbond/>
Division of Linguistics and Multilingual Studies
Nanyang Technological University
